### PR TITLE
Galactic effects & correcting new planet enviornmental effects

### PR DESCRIPTION
**Pull request by CrosswaveOmega**
This Pull Request contains the known effects of all observed Active Planet Effects, switches biome keys to the internal game names for the sake of consistency, and corrections for the temperature alternating planets.

### CODE REVIEW

File: effects/planetEffects.json
Comment (lambstream) [inline effects/planetEffects.json:1]: Please use the .editorconfig styling

File: planets/biomes.json
Comment (lambstream) [inline planets/biomes.json:1]: Please use the .editorconfig styling

File: planets/planets.json
Comment (lambstream) [inline planets/planets.json:1]: Please use the .editorconfig styling

**Comment by chatterchats** [comment]
Regarding the `planets.json` - While acknowledging that out data is also potentially mistaken, how are you determining which evnironmental effects are on the the planet? For Example, Klen Dath II (planet_001) has both intense heat and extreme cold on your version. (This is in addition to the fact that the only environmental effect it should have is Sandstorm as per the helldiversData repo by Shalzuth)

**Comment by CrosswaveOmega** [comment]
@chatterchats 
17th september, patch 01.001.100
One of the patches on September 17th internally changed the way the game handles temperature.  It's now specific to the weather settings instead of the environmental effects.  

For example, on Klen Dath II and planets that share it's settings, the extreme heat/cold modifiers is now applied whenever the time in the mission area is day/night respectively.  I built a special script to get this mapped.

**Review by lambstream — CHANGES_REQUESTED** [review]
Please update the coding style. You're using spaces instead of tabs.

**Comment by chatterchats** [comment]
> @chatterchats One of the patches on September 17th (patch 01.001.100) internally changed the way the game handles temperature. It's now specific to the weather settings instead of the environmental effects.
> 
> For example, on Klen Dath II and planets that share it's settings, the extreme heat/cold modifiers is now applied whenever the time in the mission area is day/night respectively. I built a special script to get this mapped.

@CrosswaveOmega 
Hm... That change may indicate we need to restructure, since that field has pretty much always correlated to the planet's `gameplay_modifiers`. I'll talk w/ the other folks about this to decide how we want to move forward with that

Just a brief heads-up, the json repo is formatted using 4-width tabs and not spaces, if you could please get that corrected ^^

**Comment by CrosswaveOmega** [comment]
I am attempting to get the formatting resolved in my editor, hopefully this corrects it.

File: planets/planets.json [hunk 1]
```diff
@@ -64,49 +65,50 @@
 			"pt-BR": "VANGUARDA V",
 			"de-DE": "PATHFINDER V",
 			"es-ES": "EXPLORADOR V",
-			"fr-FR": "CHERCHEVOIE V",
+			"fr-FR": "CHERCHEVOIE\u00a0V",
```
[the remark this answers is not in the document]
Comment (CrosswaveOmega) [reply planets/planets.json]: I think that was an encoding error.

File: planets/environmentals.json
[the remark this answers is not in the document]
Comment (CrosswaveOmega) [reply planets/environmentals.json:1]: the EOF should be added now.

File: planets/biomes.json
[the remark this answers is not in the document]
Comment (CrosswaveOmega) [reply planets/biomes.json:1]: Done.

**Review by chatterchats — APPROVED** [review]

**Review by dealloc — APPROVED** [review]

**Pull request merged**